### DISCUSSION TITLE
[Feature] more Homepage Categorization

### DISCUSSION THREAD

**Discussion by pyccl** [immich-app/immich#12238] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Homepage Categorization: On the homepage, clearly differentiate between photos, videos, and live photos on the left-hand side. Rename the existing "Photos" section to "Medias" (to encompass all types of media, with videos and photos displayed on the same page). Introduce new sections: "Photos" (solely for photos, excluding videos), "Videos" (exclusively for videos, without photos), and "Live Photos" (strictly for live photos, separate from videos and static photos).

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
We probably won't do this, but to add the categories on the Explore page. We want to keep the sidebar clean

**Reply by pyccl** [reply]
oh,thanks。
